src/main/java/screens/collaborative_task_scheduling/ScheduleCTController.java

```diff
@@ -16,33 +13,28 @@ public class ScheduleCTController {
 
     final ScheduleCTInputBoundary scheduleInput;
 
-    private final HashMap<String, Task> hashMap;
-
-    private final StudentUser studentUser;
+    private final Object studentUser;
 
     /**
      * Constructor for ScheduleCTController
      * @param scheduleInput - the scheduleCTInputBoundary
-     * @param hashMap - a hash map of all task ids mapped to the task object
      * @param studentUser - the current student user logged in
      */
-    public ScheduleCTController(ScheduleCTInputBoundary scheduleInput, HashMap<String, Task> hashMap, StudentUser studentUser) {
+    public ScheduleCTController(ScheduleCTInputBoundary scheduleInput, Object studentUser) {
```

Comment:
Not sure if you've updated this already or not, but you could update this to call the static method CurrentUser.getCurrentUser() in your interactor instead of the workaround you have right now